docs/proposals/2024-03-01_new_fields_for_content_analysis.md

```diff
@@ -0,0 +1,253 @@
+    "description": "Models a category used in classifying a piece of content.",
+    "type": "object",
+    "required": ["_id", "name", "classifier"],
+    "additionalProperties": false,
+    "properties": {
+      "_id": {
+        "type": "string",
+        "description": "The unique ID for this category within its classifier."
+      },
+      "classifier": {
+        "type": "string",
+        "description": "The unique identifier for the classifier that matched this category."
+      },
+      "name": {
+        "type": "string",
+        "description": "The human readable label for this category."
+      },
+      "score": {
+        "type": "number",
+        "description": "The score assigned to this category between 0 and 1, where 1 is an exact match."
+      }
+    }
+  }
+   
+```
+5. Custom Entity `src/main/resources/schema/ans/0.10.10/utils/custom_entity.json`
+```
+ {
+     "$schema": "http://json-schema.org/draft-04/schema#",
+     "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.10/utils/custom_entity.json",
+     "title": "Custom Entity",
+     "description": "Models a named custom entity (i.e. name of a person, place, or organization) used in a piece of content.",
+     "type": "object",
+     "required": ["label"],
+     "additionalProperties": false,
+     "properties": {
+       "_id": {
+         "type": "string",
+         "description": "The unique ID for this entity."
+       },
+       "custom_id": {
+         "type": "string",
+         "description": "A unique identifier for a custom-defined entity."
+       },
+       "label": {
+         "type": "string",
+         "description": "The actual string of text that was identified as a named entity."
+       },
+       "type": {
+         "type": "string",
+         "description": "A description of what the named entity is. E.g. 'organization', 'person', or 'location'."
+       },
+       "score": {
+         "type": "number",
+         "description": "The score assigned to this entity between 0 and 1, where 1 is an exact match."
+       }
+     }
+   }
+   
+  ```
+
+### Example
+In the examples below, we are using "..." where the reference URL will be updated with the latest version numbers. Otherwise, definitions are:
+
+```
+"categories": {
+      "type": "array",
+      "items": {
+        "$ref": ".../category.json"
+      },
+      "description": "A list of categories.  Categories are overall, high-level classification of what the content is about."
+    },
+"content_topics": {
+      "type": "array",
+      "items": {
+        "$ref": ".../content_topic.json"
+      },
+      "description": "A list of topics. Topics are the subjects that the content is about."
+    },
+"entities": {
+      "type": "array",
+      "items": {
+        "$ref": ".../entity.json"
+      },
+      "description": "A list of entities. Entities are proper nouns, like people, places, and organizations."
```

Comment:
I think below this line you can add the 2 fields: custom_categories and custom_entities